### PR TITLE
[WindowsIoTServices] Init.

### PR DESCRIPTION
**Pull request by LucasYao93**
## Description

+ Add test cases
+ Add examples

### CODE REVIEW

File: src/WindowsIotServices/readme.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    remove: true
+  # Rename parameter name
+  - where:
+      verb: Get|New|Update|Remove
+      subject: Service$
+      parameter-name: DeviceName$
+    set:
+      parameter-name: Name
+  # Remove cmdlet
+  - where:
```
Comment (msJinLei) [inline src/WindowsIotServices/readme.md:109]: duplicate with line 96

**Review by msJinLei — CHANGES_REQUESTED** [review]

**Comment by LucasYao93** [comment]
Hi @msJinLei,
The pipeline has a error , because the swgger refer of the DesktopVirtualization is incorrect.

Thanks

**Review by msJinLei — APPROVED** [review]

**Pull request merged**